scripts/packages/VSCodeServer/src/trees.jl

```diff
@@ -277,7 +277,7 @@ function getvariables()
         !isdefined(M, n) && continue
         Base.isdeprecated(M, n) && continue
 
-        x = getfield(M, n)
+        x = coalesce(getfield(M, n), nothing)
         x in (
```

Comment:
```suggestion
        x = getfield(M, n)
        x !== missing && x in (
```
is the correct fix here, no? Otherwise we show people's `missing` fields as `nothing` instead.

Reply:
Yep!